.github/workflows/ci.yml

```diff
@@ -140,5 +140,34 @@ jobs:
       matrix:
         python-version: [3.5, 3.6, 3.7, 3.8]
 
+  test-network:
+    name: Test Network Request
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v1
+      with:
+        python-version: 3.x
+    - uses: actions/cache@v1
+      with:
+        key: v0-${{ runner.os }}-pip-test-${{ hashFiles('setup.py') }}
+        path: ~/.cache/pip
+        restore-keys: |
+          v0-${{ runner.os }}-pip-test-
+          v0-${{ runner.os }}-pip-
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install .[test]
+        pip install https://github.com/bboe/coveralls-python/archive/github_actions.zip
```

Comment:
I don't think we need coveralls for this one test.

Reply:
```suggestion

```